src/coreclr/build.sh

```diff
@@ -526,12 +394,8 @@ handle_arguments() {
             __StaticAnalyzer=1
             ;;
 
-        stripsymbols|-stripsymbols)
-            __cmakeargs="$__cmakeargs -DSTRIP_SYMBOLS=true"
-            ;;
-
         *)
-            __UnprocessedBuildArgs+=("$1")
+            __UnprocessedBuildArgs="$__UnprocessedBuildArgs $1"
```

Comment:
We are inconsistent in using array:

* src/coreclr/tests/runtesttilstable.sh
* eng/common/cross/build-rootfs.sh
* src/libraries/Native/build-native.sh

are using this concatenation approach, so i aligned it here. Between `+=("$1")` and `="$__UnprocessedBuildArgs $1"` I wold pick latter because it is posix complaint and array is not needed for simple cases like these. in particular, posix shell does not define array by it self.

Reply:
Thank you for the explanation, it makes sense.